css-canvas-medical/styles.css

```diff
@@ -0,0 +1,186 @@
+.container {
+  max-width: 1300px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.row {
+  display: flex;
+  padding-top: 1rem;
+  padding-bottom: 2rem;
+}
+
+.header {
+  position: relative;
+  top: 1rem;
+  border-bottom: solid;
+  border-bottom-color: whitesmoke;
+}
+
+.nav {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+  overflow: hidden;
+  position: absolute;
```

Comment:
absolute will work here, but I would consider setting flex on the parent element so that you can use flexbox to position the nav as needed.